src/main/java/org/georchestra/cas/MessagesConfiguration.java

```diff
@@ -0,0 +1,66 @@
+package org.georchestra.cas;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+
+@Component
+@Order(1)
+class MessagesConfiguration implements CommandLineRunner {
```

Comment:
Why do you need to implement `CommandLineRunner` ?

Reply:
I need it to start updating message properties when application has already started